pom.xml

```diff
@@ -5,13 +5,13 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.7.3</version>
+    <version>3.0.2</version>
     <relativePath />
   </parent>
 
   <groupId>org.folio</groupId>
   <artifactId>edge-dematic</artifactId>
-  <version>1.8.0-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
Why is the upgrade version 2.0.0 instead of 1.9.0 ?

Reply:
For all spring-based modules migrated to Spring Boot v3, their versions must be upgraded to the following major version number.